### ISSUE TITLE
After editing a subscription, shipping data/items are not updated

### ISSUE THREAD

**Issue by limist** [kathyisawesome/wc-mnm-subscription-editing#33]
@helgatheviking With an MnM subscription product, I noticed that after editing such a subscription that the associated shipping data are not updated — i.e. the Items in the Shipping section of the subscription (or any orders generated) after it is edited, are all unchanged, and looks to be stuck on the original list of Items.

e.g. after upsizing a subscription from a 20-items MnM variation to a 30-items variation, and changing products and quantities, you can see the shipping section of this subscription is still showing the original smaller variation, and the original products and quantities:

![image](https://github.com/kathyisawesome/wc-mnm-subscription-editing/assets/138796/af59205c-a1a2-4bb6-a533-224034b85350)

**Comment by limist** [comment]
A related question is whether a change in the MnM variation (i.e. container upsize or downsize) will trigger a re-do of the shipping policy to apply?  If yes, is that compatible with the popular shipping plugin Table Rate Shipping?  Thanks!

**Comment by helgatheviking** [comment]
Hi Kai, just to let you know I'm on sabbatical and won't be taking these up any time soon. You might want to reach out to another dev to look into this.

I suspect shipping might not be updated without a specific request as that's usually handled/set at checkout.

**Comment by limist** [comment]
@helgatheviking Yep I was aware, just wanted to log it here for now.  Enjoy your time off!

**Comment by helgatheviking** [comment]
@limist could you test what happens to shipping when dealing with a classic variable subscription product? Core switching does go back through the checkout process so that might be where shipping is calculated.
